website/content/docs/concepts/index.mdx

```diff
@@ -1,15 +1,16 @@
 ---
 layout: docs
-page_title: Internals
+page_title: Concepts
 description: >-
-  This section covers the internals of Nomad and explains technical details of
+  This section covers the core concepts of Nomad and explains technical details of
   Nomads operation.
 ---
 
-# Nomad Internals
+# Nomad Concepts
 
-This section covers the internals of Nomad and explains the technical
-details of how Nomad functions, its architecture, and sub-systems.
+This section covers the core concepts of Nomad and explains the
+technical details of how Nomad functions, its architecture, and
+sub-systems.
 
 -> **Note:** Knowledge of Nomad internals is not
```

Comment:
I wonder if we should just remove this. The last sentence "if you're operating Nomad you should know this" does a real 180 from the first sentence! The individual `Advanced` pages all have a warning at the top already, and I don't think anybody comes to nomadproject.io thinking they have to read every page to use Nomad? I hope not!